.github/workflows/push.yml

```diff
@@ -62,11 +62,24 @@ jobs:
       - name: Install dependencies
         run: npm ci
 
+      - name: Start the development server in detach mode
+        run: |
+          npm run dev &
+          echo "Development server started..."
+
       - name: Sleep for 10s
```

Comment:
@EshaanAgg any idea why this is being used?

Reply:
This is used to start the development server so the configured plugins can make an API request to the dev server to run an introspection query to generate the GraphQL schema.